sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -100,4 +103,73 @@ export class ApplicationOfferingChangeRequestService {
       count,
     };
   }
+
+  /**
+   * Gets application offering request change list.
+   * @param locationId location id.
+   * @param paginationOptions options to execute the pagination.
+   * @param statuses list of status that need to be included in the query.
+   * @returns list of requested application offering changes.
+   */
+  async getRequestedSummary(
+    locationId: number,
+    paginationOptions: PaginationOptions,
+    statuses: ApplicationOfferingChangeRequestStatus[],
+  ): Promise<PaginatedResults<ApplicationOfferingChangeRequest>> {
+    const offeringChange = this.applicationOfferingChangeRequestRepo
+      .createQueryBuilder("applicationOfferingChangeRequest")
+      .select([
+        "applicationOfferingChangeRequest.id",
+        "applicationOfferingChangeRequest.applicationOfferingChangeRequestStatus",
+        "application.applicationNumber",
+        "currentAssessment.id",
+        "offering.studyStartDate",
+        "offering.studyEndDate",
+        "student.id",
+        "user.firstName",
+        "user.lastName",
+        "applicationOfferingChangeRequest.assessedDate",
+        "applicationOfferingChangeRequest.studentActionDate",
```

Comment:
Why do we need `assessedDate` and `studentActionDate`?

Reply:
for `date completed`, 
![image](https://github.com/bcgov/SIMS/assets/77353155/c9068830-f26e-4df4-a437-1d290b8e12b4)

Reply:
Ah, Got it. Was looking from in-progress API point of view and lost.